pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -63,6 +65,30 @@ func (e SMTAlignmentError) Type() string {
 	return ErrorSMTAlignment
 }
 
+// InPlacePodVerticalScaling represents an error due to an
+// attempt to either move a container from exclusively allocated
+// pool to shared pool or move a container from shared pool to
+// exclusively allocated pool.
+type InPlacePodVerticalScalingError struct {
+	RequestedCPUs    string
+	AllocatedCPUs    string
+	Shared2Exclusive bool
```

Comment:
nit: not sold by the name. Perhaps something more like "inconsistentCPUAllocation" ? let's figure out

Reply:
Done

Reply:
@ffromani please let me know if we can resolve this